lib/themes_on_rails/action_controller.rb

```diff
@@ -44,6 +44,8 @@ def before_filter_method(options)
             options.delete(:prepend) ? :prepend_before_filter : :before_filter
           when 4
             options.delete(:prepend) ? :prepend_before_action : :before_action
+          when 5
```

Comment:
You can probably just to `when 4, 5` instead of having a separate `when` for Rails 5